### PR TITLE
Fixes : #15478 Doc for count doesn't work

### LINKED ISSUE #15478: Doc for count doesn't work
# Improve documentation

This should be the actual query which works for count 

const { error, data, count } = await supabase
    .from('table')
    .select('*', { count: 'exact' })

whereas the doc shows this currently

(Refer https://supabase.com/docs/reference/javascript/select)

image

This is answered in this issue 
https://github.com/orgs/supabase/discussions/517

### PR DESCRIPTION
**Pull request by lalitkumawat1m**
## What kind of change does this PR introduce?

 docs update

Fixes : #15478

### CODE REVIEW

File: spec/examples/examples.yml [hunk 1]
```diff
@@ -1175,9 +1175,9 @@ functions:
           <TabPanel id="js" label="JavaScript">
 
             ```ts
-            const { count, error } = await supabase
-              .from('countries')
-              .select('*', { count: 'exact', head: true })
+            const { error, data, count } = await supabase
+              .from('table')
```
Comment (gregnr) [inline spec/examples/examples.yml:1179]: Any reason we are no longer referencing the `countries` table?

**Review by gregnr — COMMENTED** [review]
Thanks @lalitkumawat1m! Maybe instead of replacing the `count` example, we can create another example that returns `count` with data, as you do in this PR. This way we have examples for both returning data and without.

File: spec/examples/examples.yml (hunk 1 above)
Comment (gregnr) [reply spec/examples/examples.yml:1179]: Let's revert back to `countries` here since it is linked to that schema in this example. Thanks!

**Comment by awalias** [comment]
closing as stale until https://github.com/supabase/supabase/pull/15917#discussion_r1272508342 is addressed, reopen as needed

**Pull request closed without merging**